airflow/task/standard_task_runner.py

```diff
@@ -159,7 +159,7 @@ def _start_by_fork(self):
             args = parser.parse_args(self._command[1:])
 
             self.log.info("Running: %s", self._command)
-            self.log.info("Subtask %s", self._task_instance.task_id)
+            self.log.info("Task %s", self._task_instance.task_id)
```

Comment:
to me this has basically no content. i would move it to the L161 message and just put the key value task_id=%s 

wdyt?

Reply:
like the task id is only useful as an attr to identify a meaningful log event. but it is not in itself a meaningful log event.